src/js/components/Overview/LastIngestion.tsx

```diff
@@ -7,9 +7,18 @@ import { DEFAULT_TRANSLATION } from '@/constants/configConstants';
 import { useAppSelector } from '@/hooks';
 import { getDataTypeLabel } from '@/types/dataTypes';
 
+import { LastIngestionDataTypeResponse } from '@/types/lastIngestionDataTypeResponse';
+
 const LastIngestionInfo: React.FC = () => {
   const { t, i18n } = useTranslation(DEFAULT_TRANSLATION);
-  const lastEndTimesByDataType = useAppSelector((state) => state.ingestionData?.lastEndTimesByDataType) || {};
+
+  const dataTypesObject = useAppSelector((state) => state.lastIngestionData?.dataTypes) || {};
+  const dataTypesArray = Object.values(dataTypesObject);
+
+  const sortedDataTypes = dataTypesArray.sort((a, b) => a.label.localeCompare(b.label));
```

Comment:
`.sort` mutates the original value in JS, so it doesn't make sense to have two variables assigned to the same memory location here - either use `.toSorted` for not-in-place or (my preference), combine the assignments since `dataTypesArray` isn't used anywhere else:

```js
const sortedDataTypes = Object.values(dataTypesObject).sort((a, b) => a.label.localeCompare(b.label));
 ```